test/compress/keep_quoted_strict.js

```diff
@@ -0,0 +1,39 @@
+keep_quoted_strict: {
+    options = {
+        evaluate: true,
+        properties: true,
+    },
+    mangle = {
+        properties: {
+            keep_quoted: "strict",
+            reserved: ["propc", "propd"],
+        },
+    }
+    input: {
+        var a = {
+            propa: 1,
+            get propb() { return 2; },
+            propc: 3,
+            get propd() { return 4; },
+        };
+        var b = {
+            "propa": 5,
+            get "propb"() { return 6; },
+            "propc": 7,
+            get "propd"() { return 8; },
+        };
+        console.log(a.propa, a.propb, a.propc, a["propc"], a.propd, a['propd']);
+        console.log(b["propa"], b["propb"], b.propc, b["propc"], b.propd, b['propd']);
+        Object.defineProperty({}, "propa", {});
+        Object.defineProperty({}, "propc", {});
```

Comment:
Please incorporate these `Object.defineProperty` calls into the program flow with a consequential impact on `console.log` output. Move these two lines above the first `console.log` and have one statement operate on object `a` and the other on object `b`.